### ISSUE TITLE
set up continuous integration

### ISSUE THREAD

**Issue by mattkime** [jblotus/Try-Handlebars.js#36]
@jblotus do you have any experience with this? i'm lacking but it seems like a good idea.

**Comment by jblotus** [comment]
one thing I could do is set up a CI job that just builds and deploys the project once a change is merged into master.

normally CI runs tests but I don't think we have any tests other than our eyeballs for the moment.
